strict_deps/index.bzl

```diff
@@ -5,7 +5,11 @@ NpmPackage = provider()
 
 def _npm_package_aspect_impl(target, ctx):
     if (ctx.rule.kind == 'npm_link_package_store'):
-        return [NpmPackage(name=ctx.rule.attr.package)]
+        package_name = ctx.rule.attr.package
+        # TODO: Determine how to include the package field information in locally built npm package targets
+        if package_name == '':
+            package_name = target[JsInfo].npm_package_store_infos.to_list()[0].package
```

Comment:
I don't fully understand why it has access to the package name that way, but I assume the store target just links to another package??

Reply:
Yeah, the store info just contains a list of all of the npm packages that have accumulated.  But since this is the result of a a single `npm_package_store` or `npm_link_package_store`, there is only one npm module included in the stored info.